src/domain/journey/subspace/dialogs/ContributorsToggleDialog.tsx

```diff
@@ -45,12 +48,19 @@ const ContributorsToggleDialog = ({ open = false, journeyId, onClose }: Contribu
       contributorType: ContributorType.Organizations,
     }));
 
+  const virtualContributors: VirtualContributorProps[] =
+    data?.lookup.space?.account.virtualContributors.filter(vc => vc.searchVisibility === SearchVisibility.Public) ?? [];
```

Comment:
Hm, if we have external VCs then they would be part of the community but not part of its Account, right?

Reply:
Being on the subspace level, I cannot reach the VCs which are stored on space level and not on subspace level. So until there is an API endpoint to fetch all VCs in a space form a subspace or at least to get the spaceId, the only way to get them is in the account.